terror_error.go

```diff
@@ -241,13 +298,12 @@ type jsonError struct {
 // and the original global registry would be removed here.
 // This function is reserved for compatibility.
 func (e *Error) MarshalJSON() ([]byte, error) {
+	codes := strings.Split(string(e.codeText), ":")
+	class := codes[0]
 	return json.Marshal(&jsonError{
-		Error:       e.GetMsg(),
-		Description: e.description,
-		Workaround:  e.workaround,
-		RFCCode:     e.RFCCode(),
-		Line:        e.line,
-		File:        e.file,
+		Class: rfcCodeToClass[class],
```

Comment:
Can we remove this field and add a `rfccode` field instead.

Reply:
you mean rename the `Class` to `rfcCode`?

Reply:
Yes, because the `strings.Split(string(e.codeText), ":")` rule cannot be applied for all scenarios.